package.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from '@sanity/pkg-utils';
+
+export default defineConfig({
+  legacyExports: true,
+  dist: 'dist',
+  tsconfig: 'tsconfig.dist.json',
+
+  // Remove this block to enable strict export validation
```

Comment:
should we do this? 🤷

Reply:
Same as Fieldwork. Without this it gets really upset. I don't think we need to have strict export validation - At least not yet.